lib/services/consumed/autocomplete.js

```diff
@@ -0,0 +1,49 @@
+/* @flow */
+import { CompositeDisposable, Disposable } from "atom";
+
+import type { Store } from "../../store";
+
+export const autocompleteProvider = {
+  disposables: new CompositeDisposable(),
+  watchEditor: (editor: atom$TextEditor, labels: Array<string>) => {
+    return;
+  }
+};
+
```

Comment:
One thing the code base could improve on is writing comments for exported objects and functions. This would help down the line if someone wants to understand the purpose of `autocompleteProvider` for example.

What I have in mind is informal jsdoc style (so it will show up in text editor tooltips on hover):

```js
/** 
* Stores and handles cleanup of...
* Add x to this every time you do y
* 
*/
```
What do you think?

Reply:
Sure will add. I would let @aviatesk know for #1710 too.